web/components/contract/market-topics.tsx

```diff
@@ -48,26 +49,55 @@ const ContractTopicBreadcrumbs = (props: {
 }) => {
   const { contract, topics } = props
 
+  const spanRef = useRef<HTMLSpanElement>(null)
+  const [isClamped, setClamped] = useState(false)
```

Comment:
```suggestion
  const [isClamped, setClamped] = useState(true)
```

Reply:
You're right, also `handleResize` should be called once before `addEventListener` to set the initial value. I've pushed a commit to fix that.